### ISSUE TITLE
Think about a better way for resources to indicate subcategories

### ISSUE THREAD

**Issue by sen-lu** [AsylumConnect/asylum-connect-catalog#192]
Currently, we group some resource categories into supercategories (e.g., "Medical Clinic" is a subcategory under the "Medical" supercategory). See https://github.com/AsylumConnect/asylum-connect-catalog/wiki/Resource-Categories-and-Features.

Our existing resource UI of appending category icons once per supercategory does not accurately reflect resource subcategories. Even https://github.com/AsylumConnect/asylum-connect-catalog/issues/179 would not necessarily address making subcategories visible. For example, if a resource offers both Medical Clinics and Trans Health, there's no current way to see that on the site (even though we have all that data in the backend).

We should think about ways to list / make visible resource subcategories. One idea is to simply adding a "Services Offered: ..." bullet point to resource descriptions. Alternatively, we could try to improve the resource category icon UI.

**Comment by sen-lu** [comment]
@ilakumar Would love your thoughts here!

**Comment by ilakumar** [comment]
Good point @tiffanylu! I think a bullet point list of services could work. 

In terms of improving the resource category icon UI, we could have the text that appears when the user hovers be the subcategories, rather than the category. However, then we run into the issue of having a long, difficult to read list of subcategories displayed horizontally next to the icon. Another option is to customize icons to each subcategory, but then we would have too many icons on the site. Do you have any other ideas of how we might be able to tackle this issue through the resource category icon UI?
